gcp_delete_old_images_and_instances.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
```

Comment:
Could you put the rename in a separate commit? Unfortunately it makes the diff hard to read...